docs/release-notes/2.19.0.md

```diff
@@ -0,0 +1,82 @@
+      <td class="field-body">June 29, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Backup Management can now be turned on from the UI. Go to *Settings-->Advanced settings*, and you will see it in the *Technical Preview* section. We also added support of MongoDB services on-demand backup and restore. For now, it only supports ReplicaSet on S3-compatible storage.
+
+- Dashboards improvements
+
+    - There are several community-driven improvements to ProxySQL data collection, with new dashboards to expose such metrics like: *Queries Latency histograms* and `SHUNNED_REPLICATION_LAG` state.
+
+    - Fixes for Amazon Aurora service detection on [the dashboard](https://pmmdemo.percona.com/graph/d/mysql-amazonaurora/mysql-amazon-aurora-details?var-service_name=rds-aurora57-instance-1), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.
+
+- Improvements to the security of DBaaS secrets by generating passwords for installed operators. We shipped Automated operator installation in provided K8s cluster in PMM 2.18, and here we are improving its security.  (???)
+
+## New Features
+
+* [PMM-7639](https://jira.percona.com/browse/PMM-7639): Backup Management: Ability to remove Backup Location even if there are some backup artifacts on it
+* [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
+* [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet
+
+
+
+## Improvements
+* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present on dashboard Queries Latency histograms from ProxySQL
+* [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status`
+* [PMM-7584](https://jira.percona.com/browse/PMM-7584): Components Upgrade: VictoriaMetrics 1.53.1 to 1.60.0 TBD (link for VM features)
+* [PMM-8001](https://jira.percona.com/browse/PMM-8001): Better error handling when `pg_stat_monitor` has an unsupported version
+* [PMM-7659](https://jira.percona.com/browse/PMM-7659): DBaaS: Ability to specify the type of connection for DBaaS cluster during DB Cluster creation
+* [PMM-7828](https://jira.percona.com/browse/PMM-7828): DBaaS: Select Database Type by default if only one operator is installed
+* [PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Improve UX for Backup lists when Service acceptable for restore is removed
```

Comment:
I have no idea what this means?  Does this mean when there is no acceptable restore service/location available?

Reply:
```suggestion
[PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Disable 'Restore' button for backups whose service has been deleted
```

Reply:
@rnovikovP Is this ok?